src/components/pageComponents/ContentDetailText.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react'
+import {
+    makeStyles,
+  } from "@material-ui/core/";
+
+const useStyles = makeStyles((theme) => ({
+    title: {
+        color: "#2E303C",
+        textAlign: "start",
+      },
+    tag: {
+        color: "#2E303C",
+        marginRight: "8px",
+        float: "left",
+      },
+  }));
+
+export default function ContentDetailText(props){
```

Comment:
I would suggest to rename this component to `TextDetail` (analogue to Video and Audio).
Or should we rename the other two?